ArabicToRomanNumbersChanger.java

```diff
@@ -0,0 +1,131 @@
+public class ArabicToRomanNumbersChanger {
+    private StringBuilder RomanString = new StringBuilder();
```

Comment:
Use dependency injection or at least create object during creating object of ArabicToRomanNumberChanger (do it in constructor), because you create object during loading class to memory (quite early). Read something aobut eager or lazy loading